apps/re/lib/statistics/dataloader_queries.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Re.Statistics.DataloaderQueries do
+  @moduledoc """
+  Module for grouping tags queries
+  """
+  import Ecto.Query
+
+  def run_batch(_, query, :count, listings, repo_opts) do
+    listing_ids = Enum.map(listings, & &1.id)
+    default_count = 0
+
+    result =
+      query
+      |> where([p], p.listing_id in ^listing_ids)
+      |> group_by([p], p.listing_id)
+      |> select([p], {p.listing_id, count("*")})
```

Comment:
Maybe we could use `count(1)` instead of `count(*)`.

```suggestion
      |> select([p], {p.listing_id, count(1)})
```